documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,324 @@
+
+.Canceling and deleting a failed migration plan does not clean up the `populate` pods and PVC
+
+When a user cancels and deletes a failed migration plan after creating a PVC and spawning the `populate` pods, the `populate` pods and PVC are not deleted. You must delete the pods and PVC manually. link:https://issues.redhat.com/browse/MTV-678([MTV-678])
+
+.{ocp} to {ocp} migrations require the cluster version to be 4.13 or later
+
+When migrating from {ocp} to {ocp}, the version of the source provider cluster must be {ocp} version 4.13 or later. link:https://issues.redhat.com/browse/MTV-809([MTV-809])
+
+.Restricted OVA provider namespace 
+
+You can only create an OVA provider under the Forklift or MTV namespace, for use by the forklift-controller SA. This should be extended to all namespaces.link:https://issues.redhat.com/browse/MTV-681([MTV-681])
+
+.Can only scan OVA files in NFS share root folder
+
+Only the files placed under the NFS share root folder can be scanned. The fix allows placing files directly on the share and two sublevels from the root folder of the extracted OVA. link:https://issues.redhat.com/browse/MTV-680([MTV-680])
+
+.Migration from an OVA provider to a restricted namespace fails
+
+Migrating from an OVA provider to a restricted namespace fails at the step `Convert image to kubevirt`, raising an error. link:https://issues.redhat.com/browse/MTV-689([MTV-689])
+
+.Migration with multiple disks fails
+
+When migrating an OVA VM with more than one disk, the migration gets stuck in the allocate disk phase. link:https://issues.redhat.com/browse/MTV-676([MTV-676])
+
+.Hyper-V Enlightenments are not added to the converted Windows VMs
+
+Source RHV VM with Hyper-V Enlightenments fails to convert the enlightenments after converting to OpenShift Virtualization. When converting RHV VMs to Windows VMs, the Hyper-V Enlightenments do not convert, causing significant performance issues. link:https://issues.redhat.com/browse/MTV-791([MTV-791])
+
+.Operator fails on timeout when calling `provider-validate` webhook
+
+The operator can fail when it calls `provider-validate`. It is advised to change the webhooks timeout to 30s resolves this issue. link:https://issues.redhat.com/browse/MTV-718([MTV-718])
+
+.Migrated VM from OpenStack has different power states when source VM is running
+
+A VM migrated from OpenStack does not start automatically when the source VM is running. link:https://issues.redhat.com/browse/MTV-677([MTV-677])
+
+.Provider with application credential or user token authentication type remains in `staging` status
+
+When migrating a VM with an application credential or user token authentication type, the migration gets stuck in the `staging` status, without continuing and finishing in the `Ready` status. link:https://issues.redhat.com/browse/MTV-701([MTV-701])
+
+.Forklift-controller crashes when creating a plan for a VM with 2 NICs
```

Comment:
What do you mean by NIC


* network interface card (NIC)
* network interface controller (NIC)